src/components/PrimaryNav.tsx

```diff
@@ -131,6 +132,19 @@ const PrimaryNav = (): JSX.Element => {
                   )}
                 </Menu.Item>
 
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      className={`${
+                        active ? "bg-gray-100 text-gray-700" : "text-gray-900"
+                      } group flex w-full items-center rounded-md px-5 py-1.5 text-sm`}
+                      onClick={() => console.log("Token: ", currentUser?.access_token)}
+                    >
+                      Print auth JWT
```

Comment:
```suggestion
                      Print auth token
```
Think we should remove this acronym for clarity.